### PR TITLE
Feature/itrex 583

### PR DESCRIPTION
**Pull request by katja-seb**
Added the quiz endpoint at the right places to:
- save
- update
- delete 

questions and quizzes.

### CODE REVIEW

**Review by Nimeggis — DISMISSED** [review]
approved

File: src/helperScripts/validateQuiz.ts [hunk 1]
```diff
@@ -5,24 +5,30 @@ import { IQuiz } from "../types/IQuiz";
...
+): IQuiz | undefined {
     const toast: ToastService = new ToastService();
 
+    if (courseId === undefined) {
+        toast.error(i18n.t("itrex.invalidQuizCourseID"));
+        return;
+    }
+
     if (quizName == undefined || quizName.length === 0) {
+        toast.success("dskfhsjf ");
```
Comment (swjat) [inline src/helperScripts/validateQuiz.ts:20]: ?

File: src/constants/navigators/NavigationRoutes.ts [hunk 2]
```diff
@@ -55,8 +57,17 @@ export type CourseStackParamList = {
         video: IVideo;
     };
     CHAPTER_CREATE: undefined;
-    CREATE_QUIZ: { chapterId?: string | undefined; questionId?: string | undefined; quiz?: IQuiz | undefined };
-    CREATE_QUESTION: { quiz: IQuiz | undefined };
+    CREATE_QUIZ: {
+        courseId?: string;
+        chapterId?: string | undefined;
+        questionId?: string | undefined;
+        quiz?: IQuiz | undefined;
```
Comment (swjat) [inline src/constants/navigators/NavigationRoutes.ts:64]: either:   quiz?: IQuiz
or:   quiz: IQuiz | undefined

File: src/helperScripts/validateQuiz.ts (hunk 1 above)
Comment (Nimeggis) [reply src/helperScripts/validateQuiz.ts:20]: hab ich gekonnt ignoriert, sicherlich nur ne testnachricht ;)

**Review by swjat — APPROVED** [review]

**Pull request merged**